lib/config_verify/helper.go

```diff
@@ -0,0 +1,103 @@
+package config_verify
```

Comment:
please write the tests for failed scenarios

Reply:
had written test cases for failed and success scenarios.